pkg/local-storage/member/controller/scheduler/resources.go

```diff
@@ -102,7 +102,9 @@ func (r *resources) initilizeResources() {
 
 	// initialize total capacity
 	for i := range nodeList.Items {
-		r.addTotalStorage(&nodeList.Items[i])
+		if nodeList.Items[i].Status.State == apisv1alpha1.NodeStateReady {
```

Comment:
Good for me, a little question.
If a new storage nodes joined, will the scheduler discovery it and add it to the TotalStorage?

Reply:
Yeah, it can because  initilizeResources function belongs to Init func in scheduler module .

Reply:
I mean, when the scheduler itself does not need to be restarted :-)